frontend/app/src/components/StreamlitDialog/styled-components.ts

```diff
@@ -85,13 +74,13 @@ export const StyledLabel = styled.label(({ theme }) => ({
   fontSize: theme.fontSizes.sm,
 }))
 
-export const StyledSmall = styled(Small)(() => ({
+export const StyledSmall = styled(Small)(({ theme }) => ({
   display: "block",
   paddingBottom: 0,
   paddingTop: 0,
   marginBottom: 0,
   marginTop: 0,
-  lineHeight: 1.5,
+  lineHeight: theme.lineHeights.table,
```

Comment:
nit: instead of using `lineHeights.table`, maybe we should find a more common name for the line height, e.g. `small` and use the same for `alert` & `table`

Reply:
we discussed this offline and agreed that we should consolidate all three of them in one property `small: 1.5`